### ISSUE TITLE
summary4

### ISSUE THREAD

**Issue by shobhit-v** [shobhit-v/demo#87]
## Issue Description
Impact: Critical
Summary: summary4
Description: The text alternative is missing.

Standards: wcag2a, wcag2aa
Product: CustomView-TestRuns
Test Case Name: Customer_View_Testrun_TC_195952
Test Run Name: Customer_View_Testrun_TC_195952_08-12-21-08:19
Checkpoint Group: 1.1 Text Alternatives
Issue Type: Accessibility
Unit Type: Page
Test Unit: Customer_View_Testrun_Page
Digital Asset Type: Native Mobile iOS
Release: REL4
Environment: ENV4
Platform: PLFM4
Assistive technology: AT4
Checkpoint: Alternative Text (Active Images) (1.1.1.a)
Method: Manual
Tested On URL: http://abcdcomputech.dequecloud.com/
